test/blackbox-tests/test-cases/dump-gc-stat.t

```diff
@@ -1,7 +1,7 @@
-Testing the --dump-gc-stat option
```

Comment:
I was confused why this would have worked, but I think cmdliner was completing it.